src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -4396,7 +4398,7 @@ u8 sReturnEntranceGroupIndices[] = {
     0,  // ENTR_RETURN_GREAT_FAIRYS_FOUNTAIN_MAGIC
 };
 
-s32 func_80839034(PlayState* play, Player* this, CollisionPoly* poly, u32 bgId) {
+s32 Player_HandleExitsAndVoids(PlayState* play, Player* this, CollisionPoly* poly, u32 bgId) {
```

Comment:
I've noticed you've adapted use of the verb `Handle` for some of these complex logic-handling functions in player. Could you explain your thoughts on using `Handle` vs `Update` or `Process`?

Reply:
Updating to me means to simply update the state of a thing (usually every frame). `Player_UpdateHoverBoots` is updating the hover boots timer accordingly if hover boots are active.

Process to me means to take a bunch of information and parse it to do other things. `Player_ProcessSceneCollision` gets information about the scene collision from UpdateBgCheckInfo and does a whole host of other things with that data (conveyor, lights, reverb, updating floor angles, hover boots etc)

The scenes and exits function is neither of those to me. Its not "updating the voids and exits". 
You can say that its processing them, like getting the information from the floor poly that its an exit or void. But to me they are their own complex processes that get handled by that function.